sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcOptions.scala

```diff
@@ -67,4 +67,11 @@ object OrcOptions {
     "snappy" -> "SNAPPY",
     "zlib" -> "ZLIB",
     "lzo" -> "LZO")
+
+  // The extensions for ORC compression codecs
+  val extensionsForCompressionCodecNames = Map(
```

Comment:
This is moved from [object ORCFileFormat](https://github.com/apache/spark/pull/19651/files#diff-01999ccbf13e95a0ea2d223f69d8ae23L265) in `sql/hive`.